tutorials/understanding-group/index.md

```diff
@@ -0,0 +1,327 @@
+
+## Usage of the group module
+
+When the group module is enabled in a chain (say the Cosmos Hub), this means that users can create groups and submit a group proposal.
+This means that any group of users can be part of a group and vote on the group proposal. You can think of it as an enhanced multisig or DAO.
+
+Before starting, let's first review a few terminologies:
+
+* **Group Admin**: the person that creates the group is the group administrator. The group administrator is the person who can add, remove and change the group members, but does not need to be part of the group itself. Chose it wisely.
+* **[Group Policy](https://docs.cosmos.network/main/modules/group/01_concepts.html#group-policy)**: 
+* **[Decision Policy](https://docs.cosmos.network/main/modules/group/01_concepts.html#decision-policy)**: a policy that defines how the group members can vote on a proposal and how the vote outcome is calculated.
+* **Proposal**: Groups proposal works the same way as governance proposal: group members can submit proposals to the group and vote with a *Yes*, *No*, *No with Veto* and *Abstain* on proposals.
+
+In this tutorial, we will learn how to create groups, manage its members and submit and vote on a group proposal.
+After that you'll be able to create your own on-chain DAO for your own use case.
+
+## Requirements
+
+The group module has been introduced in the [v0.46.0 release](https://docs.cosmos.network/v0.46/modules/group/) of the Cosmos SDK.
+In order to follow the tutorial, you must use the binary of a chain with the group module, using a v0.46+ version of the SDK.
+For demonstration purposes, we will use `simd`, the simulation app of the Cosmos SDK.
+
+For installing `simd`, first clone the Cosmos SDK GitHub repository:
+
+```sh
+git clone https://github.com/cosmos/cosmos-sdk --depth=1 
+```
+
+Go to the cloned directory and checkout the v0.46.0 release:
+
+```sh
+cd cosmos-sdk && git checkout v0.46.0
+```
+
+Install `simd`
+
+```sh
+make install
+```
+
+Make sure the installation was successful:
+
+```sh
+simd version
+```
+
+The version number should be greater than or equal to `0.46.0`.
+
+## Configuration
+
+::: tip
+If you have used `simd` before, you might already have a `.simapp` directory in your home directory. You can skip to the next section or remove the chain directory (`rm -rf ~/.simapp`).
+:::
+
+In order to configure `simd`, you need to set the chain ID and the keyring backend.
+
+```sh
+simd config chain-id demo
+simd config keyring-backend test
+```
+
+Secondly, you need to add keys for group users; let's call them alice and bob.
+
+```sh
+simd keys add alice
+simd keys add bob
+```
+
+With `simd keys list` you can verify that our two users have been added.
+
+::: tip
+To avoid having to copy and paste the user addresses, now is a good time to export the user keys to variables that you can access and use for this tutorial.
+:::
```

Comment:
```suggestion
<HighlightBox type="tip">

To avoid having to copy and paste the user addresses, now is a good time to export the user keys to variables that you can access and use for this tutorial.

</HighlightBox>
```